icu4c/source/i18n/double-conversion-string-to-double.cpp

```diff
@@ -51,6 +51,18 @@
 // ICU PATCH: Wrap in ICU namespace
 U_NAMESPACE_BEGIN
 
+#ifdef _MSC_VER
```

Comment:
Just for reference, this change is from the upstream PR here:
https://github.com/google/double-conversion/pull/147

The `VS2012_RADIXWARN` change is nice, as it means that `IsDecimalDigitForRadix` will have optimization turned back on for Visual Studio 2013 and above. (Previously it was disabled for all MSVC).